share/spice/twitch/channel_info/twitch_channel_info.js

```diff
@@ -0,0 +1,107 @@
+(function (env) {
+    "use strict";
+    
+    var ID = 'twitch_channel_info';
+    var STREAM_ENDPOINT = '/js/spice/twitch/stream/';
+    var BASE = 'https://www.twitch.tv/';
+    
+    env.ddg_spice_twitch_channel_info = function(api_result) {
+        // Validate the response
+        if (!api_result || api_result.error) {
+            return Spice.failed('twitch_channel_info');
+        }
+        
+        var name = api_result.name.trim().toLowerCase();
+        var created_at = new Date(api_result.created_at);
+        
+        
+        // Render the response
+        Spice.add({
+            id: ID,
+            name: "Streaming",
+            data: api_result,
+            meta: {
+                sourceName: "Twitch.tv",
+                sourceUrl: BASE + name,
+                sourceIconUrl: BASE + 'favicon.ico'
+            },
+            normalize: function(item) {
+                var getLink = function(url, label){
+                    return '<a href="' + url + '" title="">' + label + '</a>';
+                }
+                return {
+                    url: BASE + item.name,
+                    title: item.name,
+                    subtitle: [
```

Comment:
To create the URL element in the subtitle please use: `{ text: "Playing " + item.game, href: "https://www.twitch.tv/directory/game/" + item.game }`

We can then remove `getLink()` as well